docs/how-to/contribute-code.rst

```diff
@@ -31,7 +31,7 @@ First thing is to ensure that you have Python 3 and pip installed. To do this ru
       $ python3 --version
       $ pip3 --version
 
-  .. group-tab:: Linux
+  .. group-tab:: Unix-like
```

Comment:
Unix-like is a near meaningless term, and one that only serves to confuse inexperienced Linux users. FreeBSD use is sufficiently niche that I'm comfortable expecting FreeBSD users to work out that their commands are close enough to Linux that they can adapt.

Reply:
...and I'm telling you that I've been working on this project for almost 8 years, and in *that entire time*, you're the first person who has asked for FreeBSD support. I haven't had *any* requests for other BSDs (or any other Unix, for that matter). 

I hate to break it to you, but non-Linux Unix platforms are *niche*. FWIW, Linux usage is pretty niche as well - but it's a *lot* more common than non-Linux Unix, and I'm not convinced that all Linux users actually know that they're running a "Unix-like" operating system. Many do - but you'd be surprised the sorts of questions we get from users. 

I'm also not wild about adding a second tab, since the instructions will be *exactly* the same, so you're adding a maintenance burden to the core team and all other contributors for the benefit of, so far, *exactly 1* user.

If FreeBSD users are going to start complaining that the best match for their usage instructions are listed under Linux, not "Unix", then I'm more than willing to offer them a 100% refund of their purchase price.